.github/template/bump-version/action.yml

```diff
@@ -14,5 +14,9 @@ runs:
   - name: Bump Version
     shell: bash
     run: |
-      pip install --no-cache-dir bump2version
+      if [[ -z "${{inputs.changes}}" ]]; then
+        echo "Missing parameter inputs.changes" 1>&2
+        exit 1
+      fi
+      pip install --no-cache-dir bump-my-version
```

Comment:
I know this is not a new issue, but I think we should pin the version of bump-my-version for the sake of reproducibility.